### PR TITLE
honeytrap: init at unstable-2020-12-10

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/honeytrap/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule {
```
Comment (SuperSandro2000) [inline pkgs/tools/security/honeytrap/default.nix:5]:
```suggestion
}:

buildGoModule {
```

**Review by SuperSandro2000 — APPROVED** [review]
other than that

**Comment by fabaff** [comment]
https://github.com/antoniomika/sish

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 119999` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>honeytrap</li>
  </ul>
</details>

**Pull request merged**